src/GraphQL/GraphQL.csproj

```diff
@@ -10,7 +10,7 @@
     <PackageTags>GraphQL;json;api</PackageTags>
     <PackageProjectUrl>https://github.com/graphql-dotnet/graphql-dotnet</PackageProjectUrl>
     <PackageIconUrl>https://raw.githubusercontent.com/graphql-dotnet/graphql-dotnet/master/assets/logo.64x64.png</PackageIconUrl>
```

Comment:
Could you please also change this to <PackageIcon> ?
```c#
<PackageIcon>logo.64x64.png</PackageIcon>
  <ItemGroup>
    <None Include="..\..\assets\logo.64x64.png" Pack="true" PackagePath="" />
  </ItemGroup>
```

Reply:
wont that conflict with https://github.com/graphql-dotnet/graphql-dotnet/pull/1478  ?

Reply:
I did not see, just answered in order to all incoming mail.

Reply:
i rebased of master. your comment should be addressed now